src/components/send-button/index.js

```diff
@@ -117,6 +118,13 @@ export default compose( [
 			publishStatus = 'publish';
 		}
 
+		const [ adLabel, setAdLabel ] = useState();
+		const [ adsWarning, setAdsWarning ] = useState();
+		const [ activeAdManageUrl, setActiveAdManageUrl ] = useState();
+		const [ activeAdManageUrlRel, setActiveAdManageUrlRel ] = useState();
+		const [ activeAdManageTarget, setActiveAdManageTarget ] = useState();
+		const newspack_ad_post_type_slug = NEWSLETTER_AD_CPT_SLUG;
```

Comment:
No need to re-assign a constant to another constant, `NEWSLETTER_AD_CPT_SLUG` can be used.